client/.eslintrc.json

```diff
@@ -1,4 +1,7 @@
 {
     "extends": ["standard", "standard-react"],
-    "parser": "babel-eslint"
+    "parser": "babel-eslint",
+    "globals": {
+      "localStorage": true
```

Comment:
Good idea here, no more needing to specify `window.localStorage`

Reply:
Thanks!